src/Configuration/Config.AzureKeyVault/src/AzureKeyVaultConfigurationProvider.cs

```diff
@@ -126,7 +126,7 @@ private async Task LoadAsync()
             // schedule a polling task only if none exists and a valid delay is specified
             if (_pollingTask == null && _reloadInterval != null)
```

Comment:
```suggestion
            if (_pollingTask == null && _reloadInterval.HasValue)
```

But this is more a matter of taste, as the C#-compiler will do this for you too.

Reply:
We generally prefer using `!= null` in our code style.